### ISSUE TITLE
Alternatively install modules as git checkout

### ISSUE THREAD

**Issue by widhalmt** [NETWAYS/icinga-installer#18]
While the installer is very useful for many setups, we have some special usecases where we would need a different approach.

For production setup I wouldn't want anything else than install modules as packages.

On the other hand, when providing support we sometimes need to replicate a specific user setup and therefore need a way to install specific versions of Icinga Web 2 modules.

I'm totally ok with adding / replacing them after the installer has run. But I don't want to invest the work if you want that feature within installer anyway. So if you think, you can't use an option to install modules as `git clone`, fine. But if you think, that might be a cool option, we can talk about how to get this into the installer in the best way.

**Comment by lbetz** [comment]
Should be possible with some code changes. And the config has to be done in the custom hiera area. But it's only for icingaweb2 modules via git, not for packages like icinga2 or icingaweb2.

**Comment by lbetz** [comment]
Yeah, code changes and a new release of icinga/icingaweb2, refs icinga/puppet-icingaweb2#301

**Comment by lbetz** [comment]
Config now includes dir /usr/local/share/icingaweb2-modules to store your own modules installed by hand.

**Comment by martialblog** [comment]
Not actively being worked on. Closing this for now
